cumulusci/robotframework/tests/salesforce/forms.robot

```diff
@@ -66,9 +66,32 @@ Lightning based form - Opportunity
 
     capture page screenshot
 
-    # ${contact_id} =       Get Current Record Id
-    # Store Session Record  Contact  ${contact_id}
-    # Validate Contact      ${contact_id}  ${first_name}  ${last_name}
+Non-lightning based form - checkbox
+    [Documentation]
+    ...  Verify that we can check and uncheck standard html checkboxes
```

Comment:
Does this relate to "standard" html checkboxes or "nonstandard" ones?

Reply:
standard html `<input type="checkbox">` elements.